### PR TITLE
added passing tabindex to button block

### LINKED ISSUE #1174: select: add tabindex for __button
Missing tabindex here very much

https://github.com/bem/bem-components/blob/v2/common.blocks/select/__button/select__button.bh.js#L8

### PR DESCRIPTION
**Pull request by qfox**
Fixes #1174

### CODE REVIEW

File: common.blocks/select/__button/select__button.bemhtml [hunk 1]
```diff
@@ -14,6 +14,7 @@ block('select').elem('button')(
             },
             id : this._select.id,
             textMaxWidth : this._select.textMaxWidth,
+            tabindex : this._select.tabindex,
```
Comment (narqo) [inline common.blocks/select/__button/select__button.bemhtml]: `tabIndex` (camel cased) to consistency with [button](https://github.com/bem/bem-components/blob/v2/common.blocks/button/button.bemhtml#L22)

Comment (qfox) [inline common.blocks/select/__button/select__button.bemhtml]: @narqo Yeah, right, sry. Fixed.

**Comment by qfox** [comment]
Do we need to add tests for this?

**Comment by qfox** [comment]
Gemini tests still failing ;-(

**Comment by slonoed** [comment]
@zxqfox it seems like sause labs issue with travis. You ran the tests locally?

**Comment by qfox** [comment]
@SLonoed well. I've got a lot errors about "Missing space after key" in dist:

```
Missing space after key at ./dist/touch-phone/touch-phone.bemdecl.js :
   836 |    },
   837 |    {
   838 |        "name": "popup",
----------------------^
   839 |        "mods": [
   840 |            {
```

I decide to run `build-gemini` and `gemini` only:

```
$ npm run gemini

> bem-components@2.0.0 gemini /home/alex/repos/bem-components
> http-server -s -p 8000 & p=$!; gemini test --reporter html --reporter flat; e=$?; kill -9 ${p}; exit ${e}

Error: Cannot launch browser firefox-v30:
[init({"takesScreenshot":true,"record-screenshots":false,"record-video":false,"screen-resolution":"1024x768","max-duration":7200,"browserName":"firefox","version":30,"platform":"Linux"})] The environment you requested was unavailable.
```

And this is similar to travis output:

```
Error: Cannot launch browser chrome-v35:
[init({"takesScreenshot":true,"record-screenshots":false,"record-video":false,"screen-resolution":"1024x768","max-duration":7200,"tunnel-identifier":"3580.3","build":"3580","tags":["CI"],"browserName":"chrome","version":35,"platform":"Linux"})] The environment you requested was unavailable.
```

Do you have some advices about fixing that?

**Comment by qfox** [comment]
I think this shouldn't block this request at all. Visible part of blocks doesn't changed. It covers only ux part and can be covered with some spec tests.

**Comment by slonoed** [comment]
To fix lint error, use this style

```
"name" : "popup",
```

You can read how to start gemini tests at https://github.com/bem/bem-components/blob/v2/README.ru.md
You need to configure and start Sauce connect.

I don't know about workflow with gemini. We need answers from repo owners about fixes without changing visible parts.

**Comment by qfox** [comment]
```
$ npm run gemini

> bem-components@2.0.0 gemini /home/alex/repos/bem-components
> http-server -s -p 8000 & p=$!; gemini test --reporter html --reporter flat; e=$?; kill -9 ${p}; exit ${e}

✓ attach enabled plain [chrome-v35]
✓ attach enabled plain [firefox-v30]
✓ attach enabled focused [chrome-v35]
...
Total: 762 Passed: 762 Failed: 0 Skipped: 0
```

**Comment by narqo** [comment]
@zxqfox Could you add tmpl-specs for this change in the same manner we have for [button](https://github.com/bem/bem-components/blob/580e7a399a455b62c967b25f530dbd1c8cacaac8/common.blocks/button/button.tmpl-specs/10-params.bemjson.js)

BTW: unfortunately, Sauce labs' instances couldn't be used for code that have came from the forks. This is why gemini tests have failed. As far as I know BEM team are thinking about the solution currently.

**Comment by qfox** [comment]
@narqo check pelase.

**Comment by tadatuta** [comment]
:+1:

File: common.blocks/select/select.tmpl-specs/80-tabindex.bemjson.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+([
+    {
+        block : 'select',
+        mods : { mode : 'radio' },
+        tabIndex : 1,
+        options : [
+            { val : 1, text : 'first' }
+        ]
+    },
+    {
```
Comment (narqo) [inline common.blocks/select/select.tmpl-specs/80-tabindex.bemjson.js:10]: I think one test w/ `tabIndex=1` should be enough here ;)

Comment (qfox) [inline common.blocks/select/select.tmpl-specs/80-tabindex.bemjson.js:10]: @narqo You think there is no difference between `mode`s? I'm thinking about dropping `options`.

Comment (narqo) [inline common.blocks/select/select.tmpl-specs/80-tabindex.bemjson.js:10]: Oh, I haven't noticed different `mode`s there, sorry.

**Comment by narqo** [comment]
Looks good, thanks

**Pull request merged**